tools/universal_file_generator_pandoc.py

```
@@ -589,11 +613,11 @@ def _add_files_to_zip(self, zf, data):
                     if content.startswith(('http://', 'https://')):
                         # Download URL
                         try:
-                            response = requests.get(content, timeout=10)
-                            if response.status_code == 200:
-                                zf.writestr(filename, response.content)
+                            response = requests.get(content, timeout=10, allow_redirects=_http_allow_redirects())
                         except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)